src/components/macroview/leftPanel/SelectElementArea.tsx

```diff
@@ -20,15 +20,20 @@ export default function SelectElementArea() {
   return (
     <VStack w="33%" h="full" bg={useMainBgColour()} spacing={0}>
       <HStack w="full" px={[2, 4, 6]} py={[2, 4]} alignItems="center">
-        <Text w="full" fontWeight="semibold" fontSize={['sm', 'md']}>
+        <Text
+          textAlign="left"
+          p={1}
+          fontWeight="semibold"
+          fontSize={['md', 'lg']}
+        >
           Elements
         </Text>
         <Input
           type="search"
-          maxW={['full']}
+          maxW="full"
           maxH="32px"
           variant="brand"
-          placeholder="Search"
+          placeholder="Search for element..."
```

Comment:
This wording kinda sounds wrong? Wouldn't it make more sense to be "Search for an element..."?

Reply:
I think the an was either too long or there may have been a discussion with Pola for this. Either way, tried adding it, seems to be fine now: [fe0679f](https://github.com/WootingKb/wooting-macros/pull/194/commits/fe0679fe2ec3d74f903e6c21cba8c6bbb67780d7)